protocols/migration/migration_reports_3_0_0_to_reports_4_0_0.py

```diff
@@ -66,6 +66,10 @@ def migrate_report_event_cancer(self, old_report_event_cancer):
             reportEventId=old_report_event_cancer.reportEventId,
         )
 
+        if new_report_event_cancer.genomicFeatureCancer.roleInCancer not in ['oncogene', 'TSG', 'both']:
+            new_report_event_cancer.genomicFeatureCancer.roleInCancer = None
```

Comment:
@Antonior26 I'm going to add a test for this part, just so we're certain it's working